audit/audit.go

```diff
@@ -0,0 +1,81 @@
+package audit
+
+import (
+	"os/user"
+
+	"go.mozilla.org/sops/logging"
+
+	"database/sql"
+
+	"fmt"
+
+	_ "github.com/lib/pq"
+	"github.com/sirupsen/logrus"
+)
+
+var log *logrus.Logger
+
+func init() {
+	log = logging.NewLogger("AUDIT")
+}
+
+var auditors []Auditor
+
+func SubmitEvent(event interface{}) {
+	for _, auditor := range auditors {
+		auditor.Handle(event)
```

Comment:
this is nice, it'll let us broadcast events to other destinations later on, like email or irc